tests/DockerfileDocs

```diff
@@ -0,0 +1,18 @@
+FROM debian:8
```

Comment:
Why not `debian:9`? jessie is really old and stretch has been out for some time. You will benefit from more up-to-date python versions at least.